Apps/RaptureAPIServer/src/main/resources/rapture/config/rapture.cfg

```diff
@@ -2,3 +2,4 @@ ServerType: "webserver"
 Categories: "alpha"
```

Comment:
Maybe just make it a minimal file with a comment in? It certainly shouldn't have FullTextSearchOn ? i.e. - "override the config here" ???